tests/conftest.py

```diff
@@ -147,3 +150,49 @@ def cmdline_test(request):
     # Import qutebrowser.app so all cmdutils.register decorators get run.
     import qutebrowser.app  # pylint: disable=unused-variable
     return request.param
+
+
+class ConfigStub:
+
+    """Stub for the config module.
+
+    Attributes:
+        data: The config data to return.
+    """
+
+    def __init__(self, signal):
+        """Constructor.
+
+        Args:
+            signal: The signal to use for self.changed.
+        """
+        self.data = {}
+        self.changed = signal
+
+    def section(self, name):
+        """Get a section from the config.
+
+        Args:
+            name: The section name to get.
+
+        Return:
```

Comment:
Whoa, good point - the [Google styleguides](https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Comments#Comments) say the same.

I'm using `Return:` (almost) everywhere in qutebrowser, but I also don't use Sphinx... Though adopting Sphinx will be the next thing after adopting pytest, and then I'll rewrite everything in that way.